internal/injector/builtin/yaml/go-main.yml

```diff
@@ -21,7 +21,38 @@ aspects:
       - prepend-statements:
           imports:
             tracer: gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer
+            profiler: gopkg.in/DataDog/dd-trace-go.v1/profiler
+            log: log
+            os: os
+            slices: slices
+            strings: strings
```

Comment:
Those two are actually not used (suppose they're left over from a previous iteration)

```suggestion
```